From: Mattia Rizzolo <mattia@debian.org>
Date: Sat, 2 Apr 2016 15:19:06 +0000
Subject: fix typo in curl.cpp: s/returing/returning/

Forwarded: https://github.com/s3fs-fuse/s3fs-fuse/pull/385
---
 src/curl.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/curl.cpp b/src/curl.cpp
index bf2f037..af8846b 100644
--- a/src/curl.cpp
+++ b/src/curl.cpp
@@ -1752,7 +1752,7 @@ int S3fsCurl::RequestPerform(void)
         // Service response codes which are >= 400 && < 500
         switch(LastResponseCode){
           case 400:
-            S3FS_PRN_INFO3("HTTP response code 400 was returned, returing EIO.");
+            S3FS_PRN_INFO3("HTTP response code 400 was returned, returning EIO.");
             S3FS_PRN_DBG("Body Text: %s", (bodydata ? bodydata->str() : ""));
             return -EIO;
 
