From: Takeshi Nakatani <ggtakec@gmail.com>
Date: Fri, 22 Mar 2019 05:22:23 +0000
Subject: Fixed configure.ac about abuses AC_CHECK_FILE

Origin: upstream, https://github.com/s3fs-fuse/s3fs-fuse/commit/df604e50fb3187048ac12057b8fac798694fda60
Bug-Debian: https://bugs.debian.org/923581
Signed-off-by: Mattia Rizzolo <mattia@debian.org>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8b83546..86a2bc8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -273,10 +273,10 @@ dnl ----------------------------------------------
 dnl short commit hash
 dnl ----------------------------------------------
 AC_CHECK_PROG([GITCMD], [git —version], [yes], [no])
-AC_CHECK_FILE([.git], [DOTGITDIR=yes], [DOTGITDIR=no])
+AS_IF([test -d .git], [DOTGITDIR=yes], [DOTGITDIR=no])
 
 AC_MSG_CHECKING([github short commit hash])
-if test “x${GITCMD}” = “xyes” -a “x${DOTGITDIR}” = “xyes”; then
+if test "x${GITCMD}" = "xyes" -a "x${DOTGITDIR}" = "xyes"; then
     GITCOMMITHASH=`git rev-parse --short HEAD`
 elif test -f default_commit_hash; then
     GITCOMMITHASH=`cat default_commit_hash`
